chrome/app.js

```diff
@@ -158,15 +165,15 @@ GmailToTrello.App.prototype.updateData = function() {
     if (self.model.trello.user !== null && self.model.trello.boards !== null) {
         self.popupView.bindData(self.model);
     }
-    self.gmailView.parsingData = false;
-    self.model.gmail = self.gmailView.parseData();
```

Comment:
Don't need this here anymore because we're parsing the data in onDetected, earlier in the process.